egs/wsj/s5/utils/validate_lang.pl

```diff
@@ -671,21 +708,21 @@ sub check_summation {
   $success1 = 1;
   if (@itset != 0) {
     $success1 = 0;
-    $exit = 1; print "--> ERROR: $lang/phones/word_boundary.txt has disambiguation symbols -- ";
+    $exit = 1; print "--> ERROR: $lang/phones/$word_boundary.txt has disambiguation symbols -- ";
     foreach (@itset) {
       print "$_ ";
     }
     print "\n";
   }
-  $success1 == 0 || print "--> $lang/phones/word_boundary.txt doesn't include disambiguation symbols\n";
```

Comment:
why sometimes a \n is printed, and other times not?

Reply:
Every printing should be followed by "\n". I did not write this part but I will fix that.